### PR TITLE
feat(templates): adds an argument to define a custom template directory

### LINKED ISSUE #6: New feature proposal: support multiple templates folder (other nodejs impls or either other platforms)
I am thinking about implementing another asyncapi codegen to support another platforms than node. Basically, a fork of node-codegen with specific templates. But I realized that all I need to change is related to handlebars templates. 

I also note that other codegens like https://github.com/swagger-api/swagger-codegen brings an argument to define a templates folder like `[(-t <template directory> | --template-dir <template directory>)]`.

If you see this approach is valid for your codegen, I can contribute a PR implementing this new argument.

Thanks!

Javier A.

### PR DESCRIPTION
**Pull request by jantoniucci**
Resolves #6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "json-schema-ref-parser": "^3.1.2",
     "lodash": "^4.17.4",
     "mkdirp": "^0.5.1",
+    "test": "^0.6.0",
```
Comment (fmvilas) [inline package.json]: Why is it needed?

Comment (jantoniucci) [reply package.json]: It was added "automagically" when I tried to run tests. It can be removed.

Comment (fmvilas) [reply package.json]: Yes, let's remove it please. It's another dependency when installing the package and will make things slower.

Comment (jantoniucci) [reply package.json]: done! IMHO is not about getting slower but about including non-sense dependencies ;)

Comment (fmvilas) [reply package.json]: Sure, this too :)

**Comment by fmvilas** [comment]
Thanks! Will publish it tomorrow.

**Comment by jantoniucci** [comment]
Your welcome! As next steps, I propose externalising templates into its own repos. Then, you can manage templates supporting  multiple platforms and technologies. Does it works for you?

**Comment by fmvilas** [comment]
Absolutely! My idea is to get rid of the generator-templates coupling so it can offer many languages/frameworks without people having code their own generator.

**Comment by fmvilas** [comment]
BTW, @jantoniucci, I just published the changes to npm.

**Pull request merged**